.github/workflows/ci.yml

```diff
@@ -142,20 +146,28 @@ jobs:
         env:
           SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_NOTIFY_BUILD }}
 
-  build-publish-master-image:
+  publish-docker-image:
     needs:
       - lint-toml-files
       - cargo-verifications
-    if: github.ref == 'refs/heads/master'
+      - publish-crates
```

Comment:
Is this needed to depend on `publish-crates`? So long as `cargo-verification` passes, then the Docker image should be built, no?